VehicleLoadingMsg load_vehicle
---
bool load_request_received
